langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiImageModelIT.java

```diff
@@ -20,11 +23,15 @@ void should_generate_image() {
                 .deploymentName(System.getenv("AZURE_OPENAI_DEPLOYMENT_NAME"))
                 .apiKey(System.getenv("AZURE_OPENAI_KEY"))
                 .logRequestsAndResponses(true)
+                .setN(1)
                 .build();
 
-        Response<Image> response = model.generate("A coffee mug in Paris, France");
+        Response<List<Image>> response = model.generate("A coffee mug in Paris, France");
 
         logger.info(response.toString());
-        assertThat(response.content().url()).isNotBlank();
+
+        Image image = response.content().get(0);
```

Comment:
nice! you can assert the list size additionally:
`assertThat(response.content()).hasSize(1);`